### PR TITLE
Updating slides 

### PR DESCRIPTION
**Pull request by jmckeePVH**
Updating my slide:

- [x] my name
- [ ] cool fact

Work for #2

### CODE REVIEW

File: _slides/14-jmckeePVH.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+My name is John.
 
 ![femalecodertocat](https://octodex.github.com/images/femalecodertocat.png)
```
Comment (Jsewar) [inline _slides/14-jmckeePVH.md:8]: Hey can you add a cool fact?

**Review by Jsewar — CHANGES_REQUESTED** [review]
Looks Good!

**Review by henrysalaspvh — COMMENTED** [review]
approved by me :)

File: _slides/14-jmckeePVH.md (hunk 1 above)
Comment (jmckeePVH) [reply _slides/14-jmckeePVH.md:8]: I am very inquisitive.

**Review by Jsewar — APPROVED** [review]
good

**Review by henrysalaspvh — APPROVED** [review]
👍

**Pull request merged**